locust/runners.py

```diff
@@ -1033,6 +1033,31 @@ def client_listener(self) -> NoReturn:
                 # emit a warning if the worker's clock seem to be out of sync with our clock
                 # if abs(time() - msg.data["time"]) > 5.0:
                 #    warnings.warn("The worker node's clock seem to be out of sync. For the statistics to be correct the different locust servers need to have synchronized clocks.")
+            elif msg.type == "locustfile":
+                logging.debug("Worker requested locust file")
+                assert self.environment.parsed_options
+                filename = (
```

Comment:
is this just in case the user didn't put .py at the end? Feels like this maybe should be done in the argument parser if possible

Reply:
The default value for locust file is "locustfile" (but gets expanded to "locustfile.py" implictly) which is why I support this. I _could_ change it to just "add .py if there is no .py at the end" which would support if someone says "locust -f module_name_without_py_at_the_end". Specifying a bare module name and then having the file transfered anyway, might be nice, but it is kinda weird, and exception logged later on (line 1048) might become confusing if locust just adds .py automatically. What do you think?